samcli/cli/command.py

```diff
@@ -21,6 +21,7 @@
     "samcli.commands.deploy",
     "samcli.commands.logs",
     "samcli.commands.publish",
+    "samcli.commands.pipeline.pipeline",
```

Comment:
Can we use [`hidden=True`](https://click.palletsprojects.com/en/7.x/api/?highlight=hidden#click.Option) instead so the command can be tested?

Reply:
Are you suggesting releasing the command but not showing it during `sam --help` ?

Reply:
Right, that way we can test the feature without changing code.

Reply:
I like this, until we have `pipeline init` ready it makes sense to keep this hidden so that we don't block any releases until we are fully ready.

Reply:
This PR is merging to non-develop branch, it won't be released. Having it released and hidden won't prevent anyone from using it which what we need to avoid at the moment as this is still in develop.

Reply:
Why not keep in a non-develop branch and the command hidden if needed?

I want to be able to test this code, and it's trivial if I don't need to make code changes.

Reply:
Oh never mind, synced offline. I thought it wasn't enabled. I'm fine with either in that case.